packages/connector/src/services/admin-api/index.ts

```diff
@@ -60,8 +55,7 @@ export class AdminApi {
   }
 
   private _getRoutes(
-    peers: PeersService,
-    accounts: AccountsService
+    accounts?: AccountsService
```

Comment:
It's optional here because it was already optional in `connector.ts`'s initialization:
```
export const start = async (accounts?: AccountsService): Promise<void> => {
```